### ISSUE TITLE
TH morphology errors

### ISSUE THREAD

**Issue by OrionRandD** [fmgilham/lisp-logo#2]
Hi,
I am getting these morphology errors now:

(diatheke nil  *temp* t -b KJV -o avmw -f internal -k Genesis:1)
Morph dispatch
cond: Unknown morphology strongMorph:TH8804!
(diatheke nil  *temp* t -b KJV -o avmw -f internal -k Genesis:2)
Morph dispatch
cond: Unknown morphology strongMorph:TH8792!
(diatheke nil  *temp* t -b KJV -o avmw -f internal -k Genesis:3)
Morph dispatch
cond: Unknown morphology strongMorph:TH8804!
(diatheke nil  *temp* t -b KJV -o avmw -f internal -k Exodus:1)
Morph dispatch
cond: Unknown morphology strongMorph:TH8802!
(diatheke nil  *temp* t -b KJV -o avmw -f internal -k Exodus:5)
Morph dispatch
cond: Unknown morphology strongMorph:TH8804!
(diatheke nil  *temp* t -b KJV -o avmw -f internal -k Exodus:5)

And the program does not open properly now in study-mode

https://imgur.com/x3D3evR

**Comment by fmgilham** [comment]
Hi, you caught the code in  flux. I just pushed my latest code and it should work.

The "unknown morphology" error is because of exactly that. The Hebrew morphology tags in the KJV module don't seem to point to anything I have access to. I changed it to a message, so execution should continue. Let me know if it doesn't.

I'm not really used to Git. I'm a dinosaur and mostly used CVS and SVN when I was a developer.

By the way, I pinned the bible-mode project. Can you access through that now?

Also you should be sure you have the necessary modules. I'm using StrongsRealGreek, MLStrong, StrongsRealHebrew, and OSHB/OSHM for Hebrew. And I also use NASB as an English text though it doesn't have morphology tags.

You can get StrongsRealGreek and StrongsRealHebrew from Xiphos:

installmgr -ri Xiphos StrongsRealGreek
installmgr -ri Xiphos StrongsRealHebrew

Get OSHB and OSHM from CrossWire

installmgr -ri CrossWire OSHB
installmgr -ri CrossWire OSHM

I'm going to make a release probably next week when I have a bit of time. There's still one major change I'd like to make but I will probably make a release before that.

**Comment by OrionRandD** [comment]
Yeah! I do have xiphos here with all these modules and much more...
I will "git pull" here to have the latest changes...
And test it again

**Comment by OrionRandD** [comment]
=> https://imgur.com/BB4XAEl.png
:)

**Comment by fmgilham** [comment]
Thanks for all your comments! I'll look into making this an emacs package. I think I saw instructions once so I should be able to figure it out.

Your screenshot is pretty cool!
